python/rmm/_lib/device_buffer.pyx

```diff
@@ -34,6 +34,18 @@ cdef class DeviceBuffer:
     def size(self):
         return self.c_obj.get()[0].size()
 
+    @property
+    def __cuda_array_interface__(self):
+        cdef dict intf = {
+            "data": self.data(),
```

Comment:
We have a property to define this already as well
```suggestion
            "data": self.ptr,
```

Reply:
Yeah there was another function below called `data()` that this was using. Though happy to choose either property or function as we see fit.